packages/browser/src/plugins/schema-filter/index.ts

```diff
@@ -77,6 +77,7 @@ export function schemaFilter(
 
   return {
     name: 'Schema Filter',
+    critical: true,
```

Comment:
Ah, now that I see the load method for schema filter and env-enrichment, I'm not sure we actually need a `critical` field at this time. Their load methods are already safe.

It's the actual importing of the modules as part of `registerPlugins` that can still fail and block.